### ISSUE TITLE
Reproducing results

### ISSUE THREAD

**Issue by Vaufreyd** [tensorboy/pytorch_Realtime_Multi-Person_Pose_Estimation#66]
Hello,

I thing I run the right way your code on my computer (with the last version of pytorch) but I did not get the same results. With VGG19, I stay stuck to mAP=0.51. I have few questions:
- Did you run it as mentionned in the main page with batch_size=100? I did not manage to run it with more than batch_size=32 (with 2x 1080  GTX Ti in my computer). Do you have any idea of this problem? 
- How many epochs did you run?
- Could you share the pytorch and other tool versions you used?

Thank you.

**Comment by mihaifieraru** [comment]
I experience the same problem with the provided code and configuration. 
Val loss seems to be converging at epoch 60 where I only get 0.51 mAP. 
I train with:
`
python train_VGG19.py --batch_size 100 --logdir {where to store tensorboardX logs}
`

Are you sure the training code train_VGG19.py is the one you used to obtain pose_model_scratch.pth ?

**Comment by yedaorman** [comment]
same question
